junit-jupiter-api/src/main/java/org/junit/jupiter/api/DisplayNameGenerator.java

```diff
@@ -195,7 +196,15 @@ default String generateDisplayNameForMethod(List<Class<?>> enclosingInstanceType
 	 */
 	static String parameterTypesAsString(Method method) {
 		Preconditions.notNull(method, "Method must not be null");
-		return '(' + ClassUtils.nullSafeToString(Class::getSimpleName, method.getParameterTypes()) + ')';
+		if (method.getParameterCount() == 0) {
+			return "()";
+		}
+		var parameterTypes = method.getParameterTypes();
+		if (parameterTypes[parameterTypes.length - 1].getName().equals("kotlin.coroutines.Continuation")) {
+			// Kotlin suspend functions have a Continuation parameter at the end that should not be included
+			parameterTypes = Arrays.copyOf(parameterTypes, parameterTypes.length - 1);
+		}
```

Comment:
:speech_balloon: This removes the `Continuation` parameter from display names because it's generated by the Kotlin compiler.